saleor/payment/gateways/adyen/utils.py

```diff
@@ -350,3 +354,70 @@ def get_payment_method_info(
         brand=brand, type="card" if payment_method == "scheme" else payment_method,
     )
     return payment_method_info
+
+
+def validate_payment_data_for_apple_pay(
```

Comment:
Maybe split Adyen utils into small files per provider?
For example `saleor/payment/gateways/adyen/utils/apple_pay.py`

Reply:
sounds good